### ISSUE TITLE
undesired paren completion in org mode.

### ISSUE THREAD

**Issue by introom** [Fuco1/smartparens#831]
In org mode, press the following key stroke:

=a<del>x=y

## Expected behavior
the text should be:
=x=y

specifically, the second = should skip over the completed = instead of inserting a new =

## Actual behavior
=x=y=

## Environment & version information

**In recent enough `smartparens` you can call `M-x sp-describe-system` to generate this report**.  Please fill manually what we could not detect automatically.  Edit the output as you see fit to protect your privacy.

- `smartparens` version: smartparens-20171201.242
- Active major-mode: org-mode
- Emacs version (`M-x emacs-version`): 25.3.1
- Spacemacs/Evil/Other starterkit (specify which)/Vanilla: vanilla
- OS: macos

**Comment by Fuco1** [comment]
I get the expected behaviour with my setup so I think this is configurable somehow.  Which means we should probably make it the default as it is indeed what the user would expect.

Do you have `smartparens-org` loaded? Try `(require 'smartparens-org)` to be sure and try again

Also, do you have `smartparens-strict-mode` enabled or not?

**Comment by introom** [comment]
hi. thanks for your quick reply.

I've enabled smartparens-org, which is automatically loaded when I open an org file.

I've enabled/disabled smartparens-strict-mode, which gives the same undesired behavior.

**Comment by Fuco1** [comment]
Oh, the github formatting messed up the first line in your report, it did not print the `<del>` part (I guess when you don't put it in the `pre` mode it thinks it's a html tag and strips it).  I can reproduce the issue when I hit backspace.

I still think this should be configurable somehow, but I'm not sure how yet.

**Comment by introom** [comment]
yes.  it's like, if the key to type is the ending paren, in the meantime the cursor is on the end of a pair, then just skip.

**Comment by Fuco1** [comment]
That's what happens for `)` and friends in strict mode and also for the `"` which is why I'm quite puzzled why it shouldn't work for `=`.
